block-storage/src/main/scala/coop/rchain/blockstorage/FileLMDBIndexBlockStore.scala

```diff
@@ -242,70 +299,67 @@ object FileLMDBIndexBlockStore {
       case t                    => UnexpectedIOStorageError(t)
     }
 
+  private def isRegularFile[F[_]: Sync](path: Path): StorageErrT[F, Boolean] =
+    EitherT(Sync[F].delay { Files.isRegularFile(path) }.attempt).leftMap[StorageError] {
+      case e: SecurityException => FileSecurityViolation(e)
+      case t                    => UnexpectedIOStorageError(t)
+    }
+
   private def listInDirectory[F[_]: Sync](dirPath: Path): StorageErrT[F, List[Path]] =
     for {
       _ <- EitherT(Sync[F].delay { dirPath.toFile.mkdir() }.attempt).leftMap[StorageError] {
             case e: SecurityException => FileSecurityViolation(e)
             case t                    => UnexpectedIOStorageError(t)
           }
-      directories <- EitherT(Sync[F].delay { Files.list(dirPath) }.attempt).leftMap[StorageError] {
-                      case e: NotDirectoryException => FileIsNotDirectory(e)
-                      case e: SecurityException     => FileSecurityViolation(e)
-                      case t                        => UnexpectedIOStorageError(t)
-                    }
-      directoriesList = directories
+      files <- EitherT(Sync[F].delay { Files.list(dirPath) }.attempt).leftMap[StorageError] {
+                case e: NotDirectoryException => FileIsNotDirectory(e)
+                case e: SecurityException     => FileSecurityViolation(e)
+                case t                        => UnexpectedIOStorageError(t)
+              }
+      filesList = files
         .collect(Collectors.toList[Path])
         .asScala
         .toList
-    } yield directoriesList
+    } yield filesList
 
-  private def listDirectories[F[_]: Sync](dirPath: Path): StorageErrT[F, List[Path]] = {
+  private def listFiles[F[_]: Sync](dirPath: Path): StorageErrT[F, List[Path]] = {
     type TEMP[A] = StorageErrT[F, A]
     for {
       inDirectoryList <- listInDirectory(dirPath)
-      directoryList   <- inDirectoryList.filterA[TEMP](isDirectory)
+      directoryList   <- inDirectoryList.filterA[TEMP](isRegularFile)
```

Comment:
What makes this TEMP?

Reply:
Accidently left this in in the last PR while playing with scalac